jobs/openvpn/templates/etc/openvpn.conf.erb

```diff
@@ -26,6 +26,14 @@ server <%= p('openvpn.server') %>
 <% p('openvpn.push_routes').each do | route | %>
 push "route <%= route %>"
 <% end %>
+<% p('openvpn.push_dns').each do |dns| %>
+# ensure we can route to the DNS server over the connection
+push "route <%= dns %> 255.255.255.255"
+push "dhcp-option DNS <%= dns %>"
+<% end %>
+<% if_p('openvpn.push_dns_search_domains') do |domains| %>
+push "dhcp-option DOMAIN-SEARCH <%= domains.join(" ") %>"
```

Comment:
Thanks for the extra info. Until there's a reason it doesn't work, your suggestion sounds good to me.